packages/appstore/src/components/app-launcher/app-launcher.scss

```diff
@@ -13,8 +13,20 @@
     }
 
     &__button {
-        height: fit-content;
         margin-left: auto;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        padding: 6px 16px;
+
+        width: 56px;
+        height: 32px;
+
+        /* button/primary-light/default */
+
+        background: rgba(255, 68, 79, 0.16);
+        border-radius: 4px;
```

Comment:
rem

Reply:
We are not using it now, and  I don't have any changes in it. better keep it like this since not the scope of this PR.

Reply:
same here